docs/config.rst

```diff
@@ -9,16 +9,79 @@ First Install
 
 During a new install, you will be prompted first for a walkthrough of the
 Installation Wizard. This wizard will help you get a few essential configuration
-options taken care of before beginning.
+options taken care of before beginning. Once done, you will be left with two files:
+
+.. describe:: config.yml
+
+    The YAML configuration was introduced in Sentry 8 and will allow you to configure
+    various core attributes. Over time this will be expanded.
+
+.. describe:: sentry.conf.py
+
+    The Python file will be loaded once all other configuration is referenced, and allows
+    you to configure various server settings as well as more complex tuning.
+
+Many settings available in ``config.yml`` will also be able to be configured in the Sentry
+UI. Declaring them in the file will generally override the dynamically configured value
+and prevent it from being changed in the UI. These same settings can also be configured via
+the ``sentry config`` CLI helper.
+
+General
+-------
+
+.. describe:: system.admin-email
+
+    Declared in ``config.yml``.
+
+    The technical contact address for this installation. This will be reported to
+    upstream to the Sentry team (as part of the Beacon), and will be the point of
+    contact for critical updates and security notifications.
+
+    ::
+
+        system.admin-email: 'admin@example.com'
+
+.. describe:: system.url-prefix
+
+    Declared in ``config.yml``.
+
+    The URL prefix in which Sentry is accessible. This will be used both for
+    referencing URLs in the UI, as well as in outbound notifications.
+
+    ::
+
+        system.url-prefix: 'https://sentry.example.com'
+
+.. describe:: system.secret-key
+
+    Declared in ``config.yml``.
+
+    A secret key used for session signing. If this becomes compromised it's
+    important to regenerate it as otherwise its much easier to hijack user
+    sessions.
+
+    ::
+
+        system.secret-key: 'a-really-long-secret-value'
+
+    To generate a new value, you can use Python:
+
+    .. code-block:: python
+
+        # $ sentry shell
```

Comment:
Yeah I was thinking about that too. Maybe ``config generate-secret-key`` and it just outputs to stdout? Could also have a prompt to fill it in for the existing file.